platform/vs/tests/bgp/test_invalid_nexthop.py

```diff
@@ -15,6 +15,8 @@ def test_InvalidNexthop(dvs, testlog):
     dvs.servers[0].runcmd("ip addr add fc00::2/126 dev eth0")
     dvs.servers[0].runcmd("ifconfig eth0 up")
 
+    dvs.runcmd("ip link set dev eth1 up")
+    dvs.runcmd("ip link set dev eth2 up")
```

Comment:
is this happening because of change in vslib/src/SwitchStateBaseHostif.cpp ? I see this file has some change in logic

Reply:
@abdosi is correct. this is due to this pr. https://github.com/Azure/sonic-sairedis/commit/0b43cab60540d38e4f533cfb36cd2dbc81d8e23b

Reply:
should do something similiar to this. https://github.com/Azure/sonic-swss/commit/aa116f41e192d971004249bea93383d526a865f6

Reply:
the correct fix should be line 13 should be changed.

Reply:
@smaheshm , what's the resolution of this? Are these comments fully resolved?  Thanks.

Reply:
@kcudnik, did you get a chance to check this one?  Thanks.